tests/test_surface_dice.py

```diff
@@ -330,21 +331,8 @@ def test_asserts(self):
         predictions_no_hot = torch.clone(predictions_hot)
         predictions_no_hot[0, :, 0, 0] = torch.tensor([2, 0])
 
-        with self.assertRaises(ValueError) as context:
-            SurfaceDiceMetric(class_thresholds=[1, 1], include_background=True)(predictions_no_hot, predictions_hot)
-        self.assertEqual("y_pred and y should be one-hot encoded.", str(context.exception))
-        with self.assertRaises(ValueError) as context:
-            SurfaceDiceMetric(class_thresholds=[1, 1], include_background=True)(predictions_hot, predictions_no_hot)
-        self.assertEqual("y_pred and y should be one-hot encoded.", str(context.exception))
-
         predictions_no_hot = predictions_no_hot.float()
         predictions_no_hot[0, :, 0, 0] = torch.tensor([0.5, 0])
```

Comment:
If these tests are remove, `predictions_no_hot` is not used and can also be remove.